### DISCUSSION TITLE
Option to create wikilink without title

### DISCUSSION THREAD

**Discussion by ibnishak** [jakewvincent/mkdnflow.nvim#96] [Ideas]
I am talking about `[[Apple]]` kinda links.

I guess a dirty way of doing it will be add a new link style as `wiki-notitle` and add an `else if` branch to [existing code](https://github.com/jakewvincent/mkdnflow.nvim/blob/69f8d9ab9d6c62cdf92a692e3ac4d21f5cb40cd6/lua/mkdnflow/links.lua#L480) as

```lua
elseif link_style == 'wiki-notitle' then
        replacement = {'[['..text..']]'}
```
I assume you would prefer to implement it as an additional config option rather than link style

A third and more general option that can cover all cases is to let users configure a  `link template` with variables - this would bring all transformations and link styles under that.

**Comment by jakewvincent** [comment]
Just now added a config option to enable this behavior. It's called `name_is_source` under the `links` table, and it's a boolean (default is `false`). I like the idea of a link template--I'll look into that.
